algorithms/linfa-trees/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 //!
-//! # Decision tree learning
+//! # Tree based learning
```

Comment:
I think `Decision tree learning` is still valid here, since extra trees and random forests are really just ensembles of decision trees